src/test/mina_automation/archive_blocks.ml

```diff
@@ -0,0 +1,23 @@
+open Executor
```

Comment:
There are a few modules around. Could you add a short doc-comment to each of them.

For instance for this one I imagine a comment to be like:

```
(**
Module to run archive_blocks utility for the given list of block files and an archive PostgreSQL database.
*)
```